test/e2e/extensions/python_workload.py

```
@@ -0,0 +1,73 @@
+    }
+    print(json.dumps(summary))
+    sys.stdout.flush()
+
+    pid = os.fork()
+    if pid == 0:
+        # Child process: detach file descriptors & hold Pandas RAM in memory for 10 minutes
+        try:
+            os.close(1)
+            os.close(2)
+        except Exception:
+            pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)